app/cp_nigeria/helpers.py

```diff
@@ -609,3 +616,91 @@ def initial_loan(self):
         total_grant = self.financial_params["grant_share"][0] * gross_capex
         amount = gross_capex - total_grant - total_equity
         return amount
+
+    def get_tariff(self):
+        return GoalSeek(self.goal_seek_helper, 0, 0.175)
+
+
+def GoalSeek(fun, goal, x0, fTol=0.0001, MaxIter=1000):
+    """
+    code taken from https://github.com/DrTol/GoalSeek_Python
+    Copyright (c) 2019 Hakan İbrahim Tol
+    """
```

Comment:
This github repo has an MIT license to use the code. I am not sure how the proper credit works. Do you know how to do this right or if this is enough?

Reply:
I think we can include this in the license file, or create a credits file.  Indicating it inline also works